.github/workflows/sonar-analysis.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         id: build_context
         uses: juliangruber/read-file-action@v1
         with:
-          path: ./target/build-context.json
+          path: ./build-context.json
```

Comment:
The `unzip` should be modified to ensure the archive is extracted to the `target` directory in the project.

Reply:
Changed it! Thanks Mike!